libkbfs/config_local.go

```diff
@@ -1213,6 +1227,48 @@ func (c *ConfigLocal) MakeDiskBlockCacheIfNotExists() error {
 	return c.resetDiskBlockCacheLocked()
 }
 
+func (c *ConfigLocal) openConfigLevelDB() (*levelDb, error) {
+	dbPath := filepath.Join(c.storageRoot, persistedConfigFolderName)
+	stor, err := storage.OpenFile(dbPath, false)
+	if err != nil {
+		return nil, err
+	}
+	return openLevelDB(stor)
+}
+
+func (c *ConfigLocal) loadSyncedTlfsLocked() (err error) {
+	syncedTlfs := make(map[tlf.ID]bool)
+	if c.IsTestMode() {
+		c.syncedTlfs = syncedTlfs
+		return nil
+	}
+	if c.storageRoot == "" {
+		return errors.New("empty storageRoot specified for non-test run")
+	}
+	ldb, err := c.openConfigLevelDB()
+	if err != nil {
+		return err
+	}
+	defer ldb.Close()
+	tlfPrefix := []byte(syncedTlfsConfigKeyPrefix)
```

Comment:
Why not just have a dedicated sync config db, rather than this key prefix thing?  Is there an advantage to having fewer levelDBs on disk if we decide to add other options later?